extensions/jsonpatch/index.md

```diff
@@ -33,10 +33,11 @@ The request URL and each Patch operation's `"path"` are complementary and
 MUST combine to target a particular resource, collection, attribute, or
 relationship.
 
-If a server supports the Patch extension, it **MUST** allow PATCH requests at
-any resource or relationship URLs that accept POST, PUT, or DELETE requests.
+If a server supports the Patch extension, it **MUST** allow Patch operations at
+any resource or relationship URLs that accept POST, PUT, PATCH, or DELETE
```

Comment:
The base spec no longer supports `PUT` for any use cases. We should probably remove it here for clarity.